### ISSUE TITLE
Any plans to upload to conda-forge?

### ISSUE THREAD

**Issue by paulbrodersen** [Penlect/rectangle-packer#18]
Hi @Penlect 

Do you have any plans to upload `rectangle-packer` to conda-forge? 

I am considering publishing [`netgraph`](https://github.com/paulbrodersen/netgraph/) there as I am slowly starting to use conda/mamba environments more than plain virtualenvs, and [someone raised a related issue](https://github.com/paulbrodersen/netgraph/issues/57). However, a package can only be published on conda-forge if all of its dependencies are already published there, and your package is the only dependency still missing. I am happy to make a pull-request with the necessary configuration files (as I need to figure out how to write them for netgraph anyway) but you should submit it for review.

Best,
Paul

**Comment by Penlect** [comment]
Hi Paul,

I have never heard of conda-forge before and I will likely not use it myself. But I am happy to help if someone can point to the steps I need to do or send a pull request.

BR,
Daniel

**Comment by paulbrodersen** [comment]
I will make a PR with the necessary configuration once I am back from my holiday in January, and link a good tutorial on how to submit the package. 

Merry Christmas! 
Paul

**Comment by paulbrodersen** [comment]
I just submitted netgraph (which depends on rectangle-packer) to conda-forge ([link to PR](https://github.com/conda-forge/staged-recipes/pull/22256)). In the process, it was pointed out to me that [rectangle-packer is already on conda-forge](https://github.com/conda-forge/rectangle-packer-feedstock). I installed both, rectangle-packer and netgraph, from conda-forge into a clean virtual environment and everything seems to work, so for me this issue is resolved.
